app/controllers/admin/volunteers.py

```diff
@@ -107,7 +109,9 @@ def updateBackgroundCheck():
         checkPassed = int(eventData['checkPassed'])
         type = eventData['bgType']
         setUserBackgroundCheck(user,type, checkPassed)
-        return ""
+        user = User.get_by_id(user)
+        createLog(user.firstName + " "+ user.lastName +"'s backgroundCheck for " +type+ " has been updated to "+ str(bool(checkPassed)))
```

Comment:
This sentence should be an action sentence like the ones before. It would be modified to "Updated user's background check to ..."